README.md

```diff
@@ -117,14 +127,20 @@ After you create a new merge request, the maintainer of the Git repository can t
 > merge request, use the includeAllVulnerabilities parameter in the frogbot-config.yml file.
 
 The Frogbot GitLab flow is as follows:
+
 1. The developer opens a merge request.
-2. The maintainer of the repository reviews the merge request and approves the scan by trigerring the manual *frogbot-scan* job.
+2. The maintainer of the repository reviews the merge request and approves the scan by triggering the manual _frogbot-scan_ job.
 3. Frogbot is then triggered by the job, it scans the merge request, and adds a comment with the scan results.
-4. Frogbot can be triggered again following new commits, by triggering the *frogbot-scan* job again.
+4. Frogbot can be triggered again following new commits, by triggering the _frogbot-scan_ job again.
    [GitLab CI Run Button](./images/gitlab-run-button.png)
 
 </details>
 
+### 👮 Security note for pull requests scanning
+
+When installing Frogbot using JFrog Pipelines, Jenkins and Azure DevOps, Frogbot will not wait for a maintainer approval before scanning newly opened pull requests. Using Frogbot with these platforms however isn't recommended for open source projects.
+When installing Frogbot using GitHub Actions and GitLab however, Frogbot will initiate the scan only after it is approved by a maintainer of the project, and therefore it is safe to be used for open source projects.
+
```

Comment:
It might also be helpful to add to the note that they should choose the maintainer carefully, after reviewing the pull request. This is so that they do not miss a vulnerable post-install script in package.json, etc. What do you think?

Reply:
Agreed @omerzi.